CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # bedrock-mongodb ChangeLog
 
+## 11.0.0 - 2025-mm-dd
+
+### Changed
+- **BREAKING**: Use mongodb driver 6.x.
+- **BREAKING**: Update error names to match bedrock best practice.
```

Comment:
Does this need to be more verbose with a list of old to new mappings?  I'm not sure how pervasive, if at all, checking for specific names might be in calling code.  Similarly may need to mention some error messages/details may have changed?